docs/tables/aws_securityhub_product.md

```diff
@@ -0,0 +1,46 @@
+# Table: aws_securityhub_product
+
+Security hub Products provides information about product integrations in Security Hub.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  product_arn,
+  company_name,
+  description
+from
+  aws_securityhub_product;
+```
+
+
+### List products provided by AWS
+
+```sql
+select
+  name,
+  company_name,
+  description
+from
+  aws_securityhub_product
+where
+  company_name = 'AWS';
+```
+
+
+### List products that send findings to security hub
```

Comment:
Is there any other information we can get about how these findings are being set, or to where?

Reply:
@cbruno10 When we subscribe to a product it checks according to AWS best practice or according to their security standards and sends findings (if there is any) to the security hub or receives findings from the hub. However, we can create insights to send/receive which type of findings we need. (For example, an insight might point out EC2 instances that are the subject of findings that detect poor security practices. An insight brings together findings from across finding providers.)